Tasks/GooglePlayPromoteV2/task.json

```diff
@@ -125,7 +125,7 @@
         }
     ],
     "execution": {
-        "Node10": {
+        "Node14": {
```

Comment:
Are you sure this will work? This is a task that runs on agent, right? We do not ship Node14 runtime with agent (and probably never will; we are currently thinking of providing node18).

Reply:
Thank you for providing more context. Like I mentioned in the description, I was unsure if the node 10 version was intended. In this case I will abandon the PR and go a different route for the CG alerts